scripts/ci-docker-images/builder-x86_64-unknown-linux-musl/Dockerfile

```diff
@@ -88,7 +85,7 @@ RUN curl --proto '=https' --tlsv1.2 -sSf \
 ENV OPENSSL_DIR=$SRC_DIR/openssl-$OPENSSL_VERSION
 
 # Install build dependencies.
-RUN apt-get update && apt-get -y install curl ninja-build cmake python3-distutils
+RUN apt-get update && apt-get -y install rsync ninja-build cmake python3-distutils
```

Comment:
why the need for rsync?

Reply:
Since version 5.3 the kernel Makefiles started to use `rsync` to install the headers:
```
mkdir -p /opt/libs; rsync -mrl --include='*/' --include='*\.h' --exclude='*' usr/include /opt/libs
```

Reply:
Ok